### PR TITLE
Move logging site extension

### PR DESCRIPTION
**Pull request by pakrym**
It's unfortunate to build logging site extension at this point in build but it requires `Microsoft.AspNetCore.App` package to generate runtime store and trim against.

Unify all site extensions under the same root `/src/SiteExtensions/`.

Depends on https://github.com/aspnet/BuildTools/pull/870.

### CODE REVIEW

**Review by natemcmaster — APPROVED** [review]
Fine for now. We will have to revisit when we start refactoring on Microsoft.AspNetCore.App and reference assemblies for 3.0

**Comment by natemcmaster** [comment]
Things should have settled down now.

**Comment by pakrym** [comment]
Blocked by https://github.com/dotnet/arcade/issues/1513

**Comment by pakrym** [comment]
@natemcmaster this seems to be close to completion. Do you want to take another look?

File: build/CodeSign.props [hunk 1]
```diff
@@ -35,6 +35,7 @@
       <FilesToSign Include="Microsoft.Extensions.Http.dll"                                  Certificate="$(AssemblySigningCertName)" Container="Microsoft.AspNetCore.App" />
       <FilesToSign Include="Microsoft.Extensions.Localization.Abstractions.dll"             Certificate="$(AssemblySigningCertName)" Container="Microsoft.AspNetCore.App" />
       <FilesToSign Include="Microsoft.Extensions.Localization.dll"                          Certificate="$(AssemblySigningCertName)" Container="Microsoft.AspNetCore.App" />
+      <FilesToSign Include="Microsoft.Extensions.Logging.AzureAppServices.dll"              Certificate="$(AssemblySigningCertName)" Container="Microsoft.AspNetCore.App" />
```
Comment (natemcmaster) [inline build/CodeSign.props]: This isn't in the shared framework.

```suggestion
      <FilesToSign Include="Microsoft.Extensions.Logging.AzureAppServices.dll"              Certificate="$(AssemblySigningCertName)" Container="Microsoft.AspNetCore.AzureAppServices.SiteExtension" />
```

**Review by natemcmaster — APPROVED** [review]
One nit. Otherwise LGTM.

**Pull request merged**